src/Feldspar/Core/Language.hs

```diff
@@ -444,14 +444,82 @@ err msg = assertMsg msg false undef
 -- FFI.hs
 --------------------------------------------------
 
-{- Omitted
-foreignImport :: ( Type (DenResult a)
-                 , SyntacticN c b
-                 , ApplySym a b FeldDomain
-                 )
-              => String -> Denotation a -> c
-foreignImport name f = sugarSymF (ForeignImport name f)
--}
+-- | Arity specific functions for FFI
+
+foreignImport0 :: (Syntactic r, Type (Internal r))
```

Comment:
Haddock produces warnings for functions without haddock which is a bit annoying. For these functions I think it's fair to add a similar haddock where the only difference is the number of arguments.

Reply:
Ok.